ViewModel/TableWithPlot/Rows/DeterministicRow.cs

```diff
@@ -20,6 +20,8 @@ public double Value
             {
                 Y = new Deterministic(value);
                 NotifyPropertyChanged();
+                ((DeterministicRow)NextRow)?.NotifyPropertyChanged("Y");
```

Comment:
Do you need to cast it?  NotifyPropertyChanged should be on the class right?

Reply:
Im not sure why i have to cast it. This was an attempt to fix some validation issues. I have made a ticket for HEC to look into this and do it better since they wrote it.
![image](https://user-images.githubusercontent.com/47757848/162498813-874f8138-c63b-400f-a718-d29e74b9efa3.png)